packages/core/admin/admin/src/pages/MarketplacePage/index.js

```diff
@@ -1,7 +1,8 @@
-import React, { useEffect, useRef } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import { useIntl } from 'react-intl';
 import { Helmet } from 'react-helmet';
 import { useQuery } from 'react-query';
+import FlexSearch from 'flexsearch';
```

Comment:
We already have [match-sorter](https://github.com/kentcdodds/match-sorter) installed in the project why not using this library instead?

What is the benefit of using flexsearch?

Reply:
ok didn't see that, I will take a look

Reply:
It definitely makes sense to use match sorter since it is already used in the project and it is half the size of flexsearch. I  updated but I am a bit surprised that this search for example still returns results 🤔  
![Screenshot 2022-03-03 at 16 13 01](https://user-images.githubusercontent.com/26598053/156596141-4294720a-8d85-46ac-aa01-a1f4a19332d8.png)
:

Reply:
That's weird, maybe it's because of the acronym matching?

Reply:
It's probably also possible to use the "Min and Max Ranking" to limit unexpected results. For example, maybe we can try to only match acronyms for the name, not the description?

Reply:
Interesting, but why would we want to match acronyms at all? I'll look into it more tomorrow, thanks for your help!